wishlist-page/wishlist.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/wishlist-page/wishlist.css">
+    <!-- <link rel="stylesheet" href="https://quantum-cl.netlify.app/css/component.css"> -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>Plant | Wishlist</title>
+</head>
+
+<body>
+
+    <!--navigation--container -->
+
+
+    <nav class="nav-bar">
+        <div class="nav--body">
+            <div class="nav--header">
+                <h2 class="nav--title">Plan<span style="color: white;">e</span>t</h2>
```

Comment:
Use consistent naming style. nav-bar can be changed to nav--bar since you are following that pattern